perses/annihilation/lambda_protocol.py

```diff
@@ -254,3 +263,42 @@ def set_alchemical_parameters(self, global_lambda,
        for parameter_name in lambda_protocol.functions:
            lambda_value = lambda_protocol.functions[parameter_name](global_lambda)
            setattr(self, parameter_name, lambda_value)
+
```

Comment:
this is just a generic engine that will take a standard sytem and turn it into a rest system.